platforms/android/build_sdk.py

```diff
@@ -380,7 +380,25 @@ def get_ndk_dir():
         return android_sdk_ndk_bundle
     return None
 
-
+def check_have_ipp_flag(cmake_file):
+    print("Checking build flag in:", cmake_file)
+    if not os.path.isfile(cmake_file):
+        print(f" ERROR: File {cmake_file} does not exist.")
+        sys.exit(1)
+        
+    with open(cmake_file, 'r') as file:
+        for line in file:
+            if line.strip().startswith("HAVE_IPP="):
```

Comment:
It makes sense to move key name to the function parameters. E.g.
check_cmake_flag_enabled(cmake_file, "HAVE_IPP")

The same function may be used for KleidiCV and other dependencies.